From 6ee37b45de8d2a8a1fe9f4db7f7a4b77f216b9ed Mon Sep 17 00:00:00 2001
From: Hans Lellelid <hans@xmpl.org>
Date: Thu, 18 Feb 2016 21:49:15 -0500
Subject: [PATCH] subprocess.mswindows does not exist in python3.5

Am hoping that the sys.platform (https://docs.python.org/2/library/sys.html#sys.platform) check is sufficient, though perhaps there was a reason this was not done initially.
---
 cmd2.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmd2.py b/cmd2.py
index 63e422f..4b11958 100755
--- a/cmd2.py
+++ b/cmd2.py
@@ -190,7 +190,7 @@ def __init__(self):
 to be installed on operating system.
 %s"""
 
-if subprocess.mswindows:
+if sys.platform == "win32":
     try:
         import win32clipboard
         def get_paste_buffer():
